### ISSUE TITLE
[Feature Request] Smart Playlists - make copy as a base for another smart playlist.

### ISSUE THREAD

**Issue by Jieiku** [ampache/ampache#2768]
I make extensive use of smart playlists. Many of them require quite a few rules, sometimes 1 smart playlist is used as a rule for another smart playlist, etc.

Often many of the options chosen are the same with just one small tweak, an enormous amount of time could be saved if Smart Playlists could be copied.

You can currently do this to make a regular playlist but not another smart playlist:

![2021-02-19_20-25](https://user-images.githubusercontent.com/106644/108583742-b0459600-72f0-11eb-9dc8-1f46bb8f132a.png)

For example if I have a smart playlist "Jake 3+ Rock" I could copy this smart playlist and make "Jake 3+ Classic Rock" simply by copying then changing the genre rule

**Comment by lachlan-00** [comment]
you can save any smartlist as a smartlist or playlist when you open them

**Comment by Jieiku** [comment]
so smartlists can be saved as smartlists now? back on 5.6.2 if you opened a smart playlist it could only be saved as a regular playlist.

I have admittedly not upgraded in a long time, but if this works now then I will try to find some freetime to make a backup and then update.

**Comment by lachlan-00** [comment]
you're right it not there in smartlists. i must have looked in the wrong spots so i will actually add that now

**Comment by lachlan-00** [comment]
ahh i know why now, in the searches you can save them but not in the smartlist itself, i'm adding that with 4f44ab401b676b0245e539e82ac049daa1859536
